controllers/fenceagentsremediation_controller.go

```diff
@@ -216,6 +229,7 @@ func (r *FenceAgentsRemediationReconciler) Reconcile(ctx context.Context, req ct
 		}
 
 		r.Log.Info("Fence Agent command was finished successfully", "Fence Agent", far.Spec.Agent, "Node name", req.Name, "Response", SuccessFAResponse)
+		commonEvents.NormalEvent(r.Recorder, node, v1alpha1.EventReasonNodeRebooted, v1alpha1.EventMessageNodeRebooted)
```

Comment:
If I recall correctly, not all fence agents return successfully only after rebooting the node. For example, I remember ipmilan does it. If this is right, the event message "Node rebooted" might be misleading.

Reply:
SGTM but I will change it to fence agent succeeded instead of stating something about the node status.